compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/TextPointerIcon.kt

```diff
@@ -18,9 +18,5 @@ package androidx.compose.foundation.text
 
 import androidx.compose.ui.input.pointer.PointerIcon
 
-// TODO Replace to PointerIcon.Text. No need for expect/actual
```

Comment:
Unresolved TODO. Please revert.

Reply:
Will be upstreamed. Didn't create and issue, as I have CL prepared( didn't push it yet)